packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,200 @@
+   */
+  link?: {|
+    accessibilityLabel?: string,
+    href: string,
+    onClick?: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+    text?: string,
+  |},
+  /**
+   * Callback fired when HelpIcon is clicked (pressed and released) with a mouse or keyboard.
+   */
+  onClick?: OnTapType,
+  /**
+   * Informational content that's displayed when the user clicks on HelpButton.
+   */
+  text: string | Element<typeof Text>,
+  /**
+   * Specifies the z-index for HelpButton's tooltip and popover to resolve any layering issues with other elements. See the [zIndex variant](https://gestalt.pinterest.systems/web/helpbutton#With-Z-index) for more details.
+   */
+  zIndex?: Indexable,
+|};
+
+/**
+ * [HelpButton](https://gestalt.pinterest.systems/web/helpbutton) is an affordance that accompanies an element on the screen. It helps describe or provide assistance on how to use the accompanying element.
+ */
+export default function HelpButton({
+  accessibilityPopoverLabel,
+  idealDirection = 'down',
+  link,
+  onClick,
+  text,
+  zIndex,
+}: Props): Node {
+  const ref = useRef(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setHovered] = useState(false);
+  const [isFocused, setFocused] = useState(false);
+  const [isActive, setActive] = useState(false);
+  const popoverId = useId();
+  const {
+    accessibilityTooltipMessage,
+    accessibilityLinkLabel: accessibilityDefaultLinkLabel,
+    accessibilityIcon,
+  } = useDefaultLabelContext('HelpButton');
+
+  useEffect(() => {
+    if (isOpen) {
+      ref?.current?.focus();
+    }
+  }, [isOpen]);
+
+  const toggleView = () => {
+    setIsOpen((currVal) => !currVal);
+    setActive((currVal) => !currVal);
+  };
+
+  const onHandleTap = (props) => {
+    toggleView();
+    if (onClick) {
+      onClick(props);
+    }
+  };
+
+  const bgIconColor = isHovered || isActive || isFocused ? 'default' : 'subtle';
+
+  const tooltipZIndex = zIndex ?? new FixedZIndex(1);
+
+  const zIndexLayer = new CompositeZIndex([new FixedZIndex(tooltipZIndex.index())]);
+
+  const textElement = typeof text === 'string' ? <Text align="center">{text}</Text> : text;
+
+  return (
+    <Flex alignItems="center" justifyContent="center" flex="none">
+      <Tooltip
+        text={accessibilityTooltipMessage}
+        idealDirection={idealDirection}
+        zIndex={tooltipZIndex}
+      >
+        <TapArea
+          accessibilityExpanded={isOpen}
+          accessibilityControls={popoverId}
+          accessibilityLabel={accessibilityTooltipMessage}
+          fullWidth={false}
+          onTap={onHandleTap}
+          ref={ref}
+          role="button"
+          rounding="circle"
+          onBlur={() => {
+            setFocused(false);
+            setActive(false);
+          }}
+          onFocus={() => setFocused(true)}
+          onMouseDown={() => setActive(true)}
```

Comment:
I think onMouseUp/onMouseDown arent needed, are covered with the onTap

Reply:
no need to specify tabIndex={0} as it's the default


![Screenshot 2023-02-14 at 2 52 05 PM](https://user-images.githubusercontent.com/10593890/218846127-4032fc31-ab01-4903-860b-039b28c0df12.png)